README.md

```diff
@@ -526,10 +527,49 @@ Supported options for INSTAGRAM_STORIES:
 | **SHARE_STICKER_IMAGE** | stickerImage   |
 | **SHARE_BACKGROUND_AND_STICKER_IMAGE** | backgroundImage, stickerImage   |
 
+### Static Values for Facebook Stories
+These can be assessed using Share.Social property
+For eg.
+```javascript
+import Share from 'react-native-share';
+
+const shareOptions = {
+    method: Share.FacebookStories.SHARE_BACKGROUND_AND_STICKER_IMAGE,
+    backgroundImage: 'http://urlto.png',
```

Comment:
Does this accept a `base64`?

Reply:
Yes it's working both with base64 data image and url.

Reply:
Maybe we can let implicit  that this param accept both `url` and `base64`.

Reply:
> Maybe we can let implicit that this param accept both `url` and `base64`.
Yes. So should me update the example source code on README file?

Reply:
Yeah, sure! I approved this before seeing your comment. 😅